scripts/demo.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
```

Comment:
provide couple example to launch the script.

./demo.sh

./demo.sh  https://example.com

./demo.sh. https://example.com:8080